main.go

```diff
@@ -394,6 +394,49 @@ func main() {
 				ctxLogger.Info("failed to order snapshot space================>", zap.Reflect("Volume ID", volumeID), zap.Reflect("Error", er11))
 			}
 			fmt.Printf("\n\n")
+		} else if choiceN == 15 {
+			fmt.Println("Enter the volume id to attach")
+			_, er11 = fmt.Scanf("%s", &volumeID)
+			volume := &provider.Volume{}
+			volume.VolumeID = volumeID
+			var instanceID string
+			fmt.Println("Enter the instance id to attach")
+			_, er11 = fmt.Scanf("%s", &instanceID)
+			volumeAttachment := &provider.VolumeAttachment{
+				Name:       volumeID + "-" + instanceID,
+				Volume:     volume,
+				InstanceID: instanceID,
+			}
+			volumeAttachmentReq := provider.VolumeAttachRequest{
+				VPCVolumeAttachment: volumeAttachment,
+			}
+			response, err := sess.Attach(volumeAttachmentReq)
+			if err != nil {
+				updateRequestID(err, requestID)
+				ctxLogger.Error("Failed to attach the volume", zap.Error(err))
+			}
+			fmt.Println("Volume attachment", response, err)
+		} else if choiceN == 16 {
+			fmt.Println("Enter the volume id to detach")
+			_, er11 = fmt.Scanf("%s", &volumeID)
```

Comment:
For detach volume ID is not required right?

Reply:
no change required